translate/storage/jsonl10n.py

```diff
@@ -246,3 +246,56 @@ def nested_set(target, path, value):
         out.write(json.dumps(units, separators=(',', ': '),
                              indent=4, ensure_ascii=False).encode(self.encoding))
         out.write(b'\n')
+
+
+class WebExtensionJsonUnit(base.TranslationUnit):
+    def __init__(self, source=None, ref=None, item=None):
+        if ref:
+            self._node = ref
+            self.notes = ref['description']
+            self._target = ref['message']
+        else:
+            self._node = OrderedDict((('message', ''), ('description', '')))
+        super(WebExtensionJsonUnit, self).__init__(source)
+
+    def setid(self, value):
+        self._id = value
+
+    def getid(self):
+        return self._id
+
+    def getlocations(self):
+        return [self.getid()]
+
+    def settarget(self, target):
+        super(WebExtensionJsonUnit, self).settarget(target)
```

Comment:
Is it actually necessary to call super here? I mean, defining `gettarget` instead looks more explicit to me.